diff --git a/common/beerocks/bwl/CMakeLists.txt b/common/beerocks/bwl/CMakeLists.txt
index a59682e7..647249bc 100644
--- a/common/beerocks/bwl/CMakeLists.txt
+++ b/common/beerocks/bwl/CMakeLists.txt
@@ -19,7 +19,7 @@ if (TARGET_PLATFORM STREQUAL "rdkb")
     # Extra libraries
     list(APPEND BWL_LIBS rt dl)
 
-elseif(TARGET_PLATFORM STREQUAL "openwrt")
+else()
 
     if (TARGET_PLATFORM_TYPE STREQUAL "ugw")
         set(BWL_TYPE "DWPAL")
@@ -103,7 +103,9 @@ elseif(BWL_TYPE STREQUAL "NL80211")
         ${PLATFORM_STAGING_DIR}/usr/lib
     )
 
-    list(APPEND BWL_LIBS nl-tiny)
+    find_package(nl-3 REQUIRED)
+    find_package(nl-genl-3 REQUIRED)
+    list(APPEND BWL_LIBS nl-3 nl-genl-3 wpa_client)
 
 elseif(BWL_TYPE STREQUAL "DUMMY")
 
