diff --git a/recipes-ccsp/ccsp/ccsp-snmp-pa.bb b/recipes-ccsp/ccsp/ccsp-snmp-pa.bb
index 430cc0c..5ff8c27 100644
--- a/recipes-ccsp/ccsp/ccsp-snmp-pa.bb
+++ b/recipes-ccsp/ccsp/ccsp-snmp-pa.bb
@@ -23,7 +23,7 @@ CFLAGS_append = " \
     -I=${includedir}/ccsp \
     -I${STAGING_INCDIR}/syscfg \
     "
-CFLAGS_append = "${@base_contains("DISTRO_FEATURES", "snmppa", " -DSNMP_PA_ENABLE ", " ", d)}"
+CFLAGS_append = "${@bb.utils.contains("DISTRO_FEATURES", "snmppa", " -DSNMP_PA_ENABLE ", " ", d)}"
 
 LDFLAGS_append = " \
     -ldbus-1 \
@@ -31,7 +31,7 @@ LDFLAGS_append = " \
     "
 
 do_install_append () {
-	if ${@base_contains('DISTRO_FEATURES', 'snmppa', 'true', 'false', d)}; then
+	if ${@bb.utils.contains('DISTRO_FEATURES', 'snmppa', 'true', 'false', d)}; then
 		install -d ${D}/etc
 		touch ${D}/etc/SNMP_PA_ENABLE
 		# Config files and scripts
@@ -50,19 +50,19 @@ do_install_append () {
 PACKAGES += "${PN}-ccsp"
 
 FILES_${PN}-ccsp = " \
-	${@base_contains("DISTRO_FEATURES", "snmppa", "/etc/* ", " ", d)} \
-	${@base_contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/snmpd.conf ", " ", d)} \
-	${@base_contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/run_snmpd.sh ", " ", d)} \
-	${@base_contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/run_subagent.sh ", " ", d)} \
-	${@base_contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/Ccsp*.xml ", " ", d)} \
-	${@base_contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/SELFHEAL-DEVICE-MIB.xml ", " ", d)} \
-	${@base_contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/XOPS-DEVICE-MGMT-MIB.xml ", " ", d)} \
-	${@base_contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/DEVICE-WEBPA-MIB.xml ", " ", d)} \
+	${@bb.utils.contains("DISTRO_FEATURES", "snmppa", "/etc/* ", " ", d)} \
+	${@bb.utils.contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/snmpd.conf ", " ", d)} \
+	${@bb.utils.contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/run_snmpd.sh ", " ", d)} \
+	${@bb.utils.contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/run_subagent.sh ", " ", d)} \
+	${@bb.utils.contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/Ccsp*.xml ", " ", d)} \
+	${@bb.utils.contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/SELFHEAL-DEVICE-MIB.xml ", " ", d)} \
+	${@bb.utils.contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/XOPS-DEVICE-MGMT-MIB.xml ", " ", d)} \
+	${@bb.utils.contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/DEVICE-WEBPA-MIB.xml ", " ", d)} \
 "
 
 FILES_${PN}-dbg = " \
-	${@base_contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/.debug ", " ", d)} \
-	${@base_contains("DISTRO_FEATURES", "snmppa", "${prefix}/src/debug ", " ", d)} \
-	${@base_contains("DISTRO_FEATURES", "snmppa", "${bindir}/.debug ", " ", d)} \
-	${@base_contains("DISTRO_FEATURES", "snmppa", "${libdir}/.debug ", " ", d)} \
+	${@bb.utils.contains("DISTRO_FEATURES", "snmppa", "${prefix}/ccsp/snmp/.debug ", " ", d)} \
+	${@bb.utils.contains("DISTRO_FEATURES", "snmppa", "${prefix}/src/debug ", " ", d)} \
+	${@bb.utils.contains("DISTRO_FEATURES", "snmppa", "${bindir}/.debug ", " ", d)} \
+	${@bb.utils.contains("DISTRO_FEATURES", "snmppa", "${libdir}/.debug ", " ", d)} \
 "
diff --git a/recipes-ccsp/util/test-and-diagnostic.bb b/recipes-ccsp/util/test-and-diagnostic.bb
index c3069c6..18f0dec 100644
--- a/recipes-ccsp/util/test-and-diagnostic.bb
+++ b/recipes-ccsp/util/test-and-diagnostic.bb
@@ -31,9 +31,9 @@ LDFLAGS_append = " \
     "
 
 # Fan & Thermal Control Feature
-HASTHERMAL = "${@base_contains('DISTRO_FEATURES', 'thermalctrl', 'true', 'false', d)}"
-CFLAGS_append += "${@base_contains('DISTRO_FEATURES', 'thermalctrl','-DFAN_THERMAL_CTR','',d)}"
-LDFLAGS_append += "${@base_contains('DISTRO_FEATURES', 'thermalctrl',' -lhal_platform','',d)}"
+HASTHERMAL = "${@bb.utils.contains('DISTRO_FEATURES', 'thermalctrl', 'true', 'false', d)}"
+CFLAGS_append += "${@bb.utils.contains('DISTRO_FEATURES', 'thermalctrl','-DFAN_THERMAL_CTR','',d)}"
+LDFLAGS_append += "${@bb.utils.contains('DISTRO_FEATURES', 'thermalctrl',' -lhal_platform','',d)}"
 
 
 do_install_append () {
diff --git a/recipes-rdkb/sysint-broadband/sysint-broadband.bb b/recipes-rdkb/sysint-broadband/sysint-broadband.bb
index 506613b..7363333 100644
--- a/recipes-rdkb/sysint-broadband/sysint-broadband.bb
+++ b/recipes-rdkb/sysint-broadband/sysint-broadband.bb
@@ -43,7 +43,7 @@ do_install() {
         fi
 	install -m 0755 ${S}/*.sh ${D}/rdklogger/
         
-        if ${@base_contains('DISTRO_FEATURES', 'snmppa', 'false', 'true', d)}; then
+        if ${@bb.utils.contains('DISTRO_FEATURES', 'snmppa', 'false', 'true', d)}; then
              rm -f ${D}/rdklogger/handlesnmpv3.sh
         fi
 
@@ -84,7 +84,7 @@ do_install() {
         install -m 0644 ${S}/etc/telemetry2_0.properties ${D}${sysconfdir}
 	install -m 755 ${S}/log_timestamp.sh ${D}${sysconfdir}
    	install -m 755 ${S}/postwanstatusevent.sh ${D}${base_libdir}/rdk
-        if ${@base_contains('DISTRO_FEATURES', 'snmppa', 'true', 'false', d)}; then
+        if ${@bb.utils.contains('DISTRO_FEATURES', 'snmppa', 'true', 'false', d)}; then
              install -m 755 ${S}/handlesnmpv3.sh ${D}${base_libdir}/rdk
         fi
 }
